src/test/java/CalculatorTest.java

```diff
@@ -0,0 +1,67 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+public class CalculatorTest {
+    private final Calculator calculator = new Calculator();
+
+    private Digits createDigits(String text) {
+        return Digits.of(text);
+    }
+
+    private Operators createOperators(String text) {
+        return Operators.of(text);
+    }
+
+    @Test
+    public void 덧셈() {
+        String text = "1 + 1";
+        int result = calculator.calculate(createDigits(text), createOperators(text));
```

Comment:
```suggestion
        int result = calculator.calculate("1 + 1");
```

텍스트 자체를 인자로 받는 메서드를 오버로딩해도 좋지 않을까?